dialogue-annotations-processor/build.gradle

```diff
@@ -30,13 +30,16 @@ dependencies {
 
     annotationProcessor 'com.google.auto.service:auto-service'
     compileOnly 'com.google.auto.service:auto-service'
+    annotationProcessor 'net.ltgt.gradle.incap:incap-processor'
+    compileOnly 'net.ltgt.gradle.incap:incap'
     annotationProcessor 'org.immutables:value'
     compileOnly 'org.immutables:value::annotations'
     annotationProcessor 'org.derive4j:derive4j'
     compileOnly 'org.derive4j:derive4j-annotation'
 
     testAnnotationProcessor 'org.immutables:value'
     testCompileOnly 'org.immutables:value::annotations'
+    testCompileOnly 'net.ltgt.gradle.incap:incap'
```

Comment:
Necessary because the `IncrementalAnnotationProcessor` annotation has class retention. This is acceptable because downstream consumers do not compile against the annotation processor, see https://github.com/tbroyer/gradle-incap-helper#usage.